src/static/js/init/main.js

```diff
@@ -1,2 +1,62 @@
 import GLOBAL from '../global/global';
-import './data';
\ No newline at end of file
+import Viewer from '../components/viewer';
+import './data';
+
+
+function initViewer() {
+  if (typeof Viewer === 'undefined') {
+    return false;
+  }
+
+  const common = {};
+
+  Array.prototype.forEach.call(document.getElementsByClassName('JS-Viewer'), function(elem) {
+    if (!elem.classList.contains("JS-Viewer-Ready")) {
+      const elParams = GLOBAL.parseData(elem.dataset.viewerParams);
+      const params = Object.assign({}, common, elParams);
+
```

Comment:
add console.log(Viewer)
add console.log(elem)
add console.log(params)

what inside them?

Reply:
console.log(Viewer) - shows the function, some details
console.log(elem) - shows 3 div with JS clases
console.log(params) - shows 3 objects {}